### ISSUE TITLE
canPerformAction needs to have the message argument

### ISSUE THREAD

**Issue by yasserf** [demux/deepstream-passport#3]
canPerformAction requires the username, message and callback arguments. We should keep this API consistent so that we can mixin other plugins for permission handling going forward

**Comment by demux** [comment]
Yeah, you're probably right. I think the callback should be the last argument, and the user argument needs to be passed to these as well. I'll just squeeze it in there before callback.
